### PR TITLE
Wire up builds to Appveyor

### PR DESCRIPTION
**Pull request by jclin**
@impurator @rockhymas @jameswelle 

This library is now wired up to Appveyor, and did some related cleanup.

[Delivers #139032017]

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 1.0.{build}
```
Comment (jameswelle) [inline appveyor.yml]: We can do the following I think.

- make the build number come from the AppVeyor build number
- use the 1st class build/pack functionality in AppVeyor

We some some of this in System.Doubles.

https://github.com/articulate/doubles/blob/master/appveyor.yml

Comment (jclin) [inline appveyor.yml]: Sure, but just to make sure I understood correctly, you're OK with the assembly version changing from 2.9.x. -> 1.0.0.x?

Comment (jameswelle) [inline appveyor.yml]: No @jclin - you should be able to set it up so AppVeyor build `2.9.x` creates package version `2.9.x`, right?

**Comment by jclin** [comment]
Updated to patch the assembly version from the build's.

**Pull request merged**